advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/supported_versions.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Product compatibility"
+description: Verify that your Oracle or Postgres database version is compatible with EDB Data Migration Service.
+---
+
+<div id="supported_versions" class="registered_link"></div>
+
```

Comment:
The initial/public preview version of the data migration service is only available for use in the AWS BAH (EDB hosted) environment.  We need to make that clear to users. I suggest adding a section here like the following:

## Supported EDB Postgres AI environments
The Data Migration Service is currently a public preview capability that is only available in the EDB Hosted EDB Postgres AI Cloud Service deployment on AWS.

Reply:
@ksnavely @jperozoa What do you think about adding the note about which EDB Postgres AI Cloud Service environment the Data Migration Service is available in?

Reply:
I went ahead and added the section at the top of the page for now. I can remove if we don't want it later. I think it adds some value personally.